osu.Game/Users/ExtendedUserPanel.cs

```diff
@@ -97,6 +98,13 @@ protected FillFlowContainer CreateStatusMessage(bool rightAlignedChildren)
             return statusContainer;
         }
 
+        protected FillFlowContainer CreateGroupBadges()
```

Comment:
I don't think this is required? You can just add it directly in the `UserListPanel`